CSS/style.css

```diff
@@ -631,3 +714,303 @@ section.contact_me_mobile > hr {
   margin-top: 1rem;
   margin-left: 30%;
 }
+
+.desktop-section header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  list-style: none;
+  padding-left: 10rem;
+  padding-right: 8rem;
+}
+
+.desktop-section header ul {
+  display: flex;
+  width: 16rem;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  padding: 20px;
+}
+
+.desktop-section header ul li a {
+  display: flex;
+  text-decoration: none;
+  color: #344563;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+}
+
+.desktop-landing {
+  display: grid;
+  width: 100%;
+  height: auto;
+  background-color: white;
+  padding-left: 3rem;
+  padding-right: 1rem;
+  overflow-x: auto;
+  border-bottom-left-radius: 100px;
+  grid-template-columns: repeat(1, 1fr);
+}
+
+.desktop-landing-content {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 721.46px;
+  background-image: url("../Images/landing_desktop.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top;
+}
+
+.desktop-landing-content h1 {
+  width: 40%;
+  margin-top: 10rem;
+  margin-left: 12rem;
+  align-items: center;
+  letter-spacing: 0.37px;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 60px;
+  margin-bottom: 1rem;
+  color: var(--h1-color);
+}
+
+.desktop-landing-content .wrapper {
+  max-width: 15ch;
+  white-space: nowrap;
+}
+
+.desktop-landing-content .lets-con-desk-txt {
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif, Courier, monospace;
+  min-width: 117px;
+  max-height: 24px;
+  color: #7f8cff;
+  text-align: left;
+  margin-left: 12rem;
+  margin-bottom: 0.5rem;
+}
+
+.desktop-landing-content .lets-con-desk-icon {
+  width: 11rem;
+  color: #7f8cff;
+  text-align: left;
+  margin-left: 12rem;
+}
+
+.desktop-landing-content .lets-con-desk-icon ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 170px;
+  list-style: none;
+}
+
+.desktop-card-section {
+  display: grid;
+  background-color: #f7f7f9;
+  width: 100%;
+  height: 100%;
+  margin-top: 50px;
+}
+
+.desktop-card {
+  display: flex;
+  border-radius: 24px;
+  justify-self: center;
+  margin-bottom: 5rem;
+  margin-top: 5rem;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  height: 496px;
+  gap: 1rem;
+}
```

Comment:
###  Sorry this wasn't mentioned before so I set it as **[OPTIONAL]** changes, still its highly recommended since the website should be responsive in all different screen sizes 💪🏼 

While you improved the project card content spacing has been asked by the previous reviewer 👍🏼 I think there is still room for improvement since this project is a good practice for learning web responsiveness 🧑‍🎓, currently the project card has a fixed width that has been set to the class `desk_main_img3` and because of that, the card is overlapping over the window

- A suggested fix could be setting another `media-query` to update the `flex-direction` from **"row"** to a **column** in the class `.desktop-card` so it looks similar to the project card in the mobile version.

-  Or the best suggestion is to avoid setting a background image to the class `.desk_main_img3` and also removing the property  `width: 544px`, since the projected image is content and it's not a decoration image 👍🏼.

More information on [When to Use HTML <img> Tag and CSS background-image Property](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html)

### Your Design
![image](https://user-images.githubusercontent.com/68030297/229241827-c769128b-2bb4-4bf1-97dc-7a9b2249e0a2.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/229241919-142ad34d-613d-44fb-97ac-2916fe5795d1.png)